Pacifique78/ADC
# #168445511 Feature get all mentors from the database

#### What does this PR do?
Implements logics and add tests for, get all mentors from the database 

#### Description of Task to be completed
The logic is I have first to retrieve all mentors from the database and return them but without some properties like passwords. 

#### How should this be manually tested?
Use the route provided to see if the desired output is similar to the output generated and Travis tests must pass. 

#### What are the relevant pivotal tracker stories?
#168445511

#### Screenshots (if appropriate)
![Screenshot (44)](https://user-images.githubusercontent.com/50268181/64751819-b93c1080-d51d-11e9-99c4-ef7cc6676768.png)
![Screenshot (45)](https://user-images.githubusercontent.com/50268181/64751820-b9d4a700-d51d-11e9-8df2-b76818b4557f.png)
![Screenshot (46)](https://user-images.githubusercontent.com/50268181/64751821-b9d4a700-d51d-11e9-8f5d-8dde413a18b7.png)
